templates/bake/Template/add.twig

```diff
@@ -19,4 +19,4 @@
  * @var \{{ entityClass }} ${{ singularVar }}
  */
 ?>
-{% element 'form' %}
+{{ element('form') -}}
```

Comment:
Whitespacing consistency should be maintained between the add and edit templates.

Here trailing whitespace is removed using `-}}` while it's not in edit.twig.

Reply:
It shouldn't be needed. I accidentally added a newline and was trying to fix it. Thought I updated both.